checkout/sample/src/main/java/com/worldpay/access/checkout/MainActivity.kt

```diff
@@ -42,12 +40,7 @@ class MainActivity : AppCompatActivity(), CardListener, SessionResponseListener
         card.cardListener = this
         card.cardValidator = AccessCheckoutCardValidator()
 
-        CardConfigurationClientFactory.createClient().getCardConfiguration(getBaseUrl(), object : Callback<CardConfiguration> {
-            override fun onResponse(error: Exception?, response: CardConfiguration?) {
-                response?.let { card.cardValidator = AccessCheckoutCardValidator(it) }
-                error?.let { debugLog("MainActivity", "Error while fetching card configuration: $it")  }
-            }
-        })
+        CardConfigurationFactory.getRemoteCardValidatorConfiguration(card, getBaseUrl())
```

Comment:
Just a thought... would it look better if we had a companion object in `CardConfiguration` and we held this utility method inside of there instead of the factory. So it would look like:

`CardConfiguration.getRemoteCardConfiguration(card, getBaseUrl())`

It's a little tricky also because this function is responsible for getting the remote configuration as well as setting the returned config on the `Card` object, therefore it may be a little misleading just calling it `getRemote...()`

Reply:
Also apply change to MainActivity.java